!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  center_limit: 10
  center_origin: 0.0
  focus_max_speed: 3.0
  fovea_radius: 0.05
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      center_limit: 10.0
      center_origin: 0.0
      focus_max_speed: 3.0
      fovea_radius: 0.05
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      keep_focus_centered: true
      left_below: false
      name: Default
      near_below: true
      near_limit: -10.0
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      type: ''
      wait_stable: 0.5
    state: []
  keep_focus_centered: true
  left_below: false
  near_below: true
  near_limit: -10
  wait_stable: 0.5
state: []
